stytra/core.py

```diff
@@ -141,7 +140,7 @@ class Stytra:
 
     """
 
-    def __init__(self, recording=None, exec=True, app=None, **kwargs):
+    def __init__(self, exec=True, app=None, **kwargs):
```

Comment:
I do agree that recording does not merit such a special place since we already dispatch on the presence of various keys in kwargs. This dispatching as a whole doesn't feel quite right, but to overcome it we would probably need an elaborate config system like [Hydra](https://hydra.cc/)